### ISSUE TITLE
Do not add note to cache for action

### ISSUE THREAD

**Issue by bikerunner** [Thricehappy/EveScoutRescue#217]
The current implementation adds the action as text to the cache. This duplicates the action list in reverse order.

*Changes:*
* do not add sow and text actions as note text to a cache
* only add text notes to the cache notes

**Comment by Klensor** [comment]
Ok, so I had a think about this. We can show only tends with notes, but these are also shown in the action list - is there a reason to duplicate any of it?

I got to thinking about what else could go in that 'notes' space - I was thinking it would be cool to have some stats - for example, length of time since cache was sown, most times sown / tended etc.

Then I thought (perhaps a little too radically) wouldn't it be cool to have different levels of system/cache - for example, we could sort the systems table and then say that the top 5-10% was designated as "Rare" systems (which could mean extra cash for tending, or maybe just stats bragging rights - obviously with a thought to people abusing such a thing), with the other systems labelled "Uncommon", "Common" etc based on their history. Thoughts?

**Comment by Thricehappy** [comment]
@chigaze Can you respond to this?

**Comment by chigaze** [comment]
If I'm understanding this correctly, the proposal would be to no longer have a Notes section above the History section and only have the notes for the actions. This makes sense to me as they duplicate the history and they make more sense in the context of the history as older ones may no longer be relevant if a cache expired and was moved. 

While I like the idea of identifying systems by how often we've been there, or perhaps by how long it's been since we've been there, I would like to see the system level notes, currently a tiny note icon and plus icon beside the system class, better exposed. I doubt many asides from myself and Thrice know they're there.

I don't have a way to search for those but an example of one is in J111159 where I added a note indicating I set a 6 month no sow and why.
